packages/site/src/main.css

```diff
@@ -40,51 +40,55 @@ a:visited {
   color: var(--color-text--secondary);
 }
 
-custom-elements {
+custom-elements,
+section[role="tabpanel"] {
```

Comment:
@scotttjob I tacked the selector for these changes onto your existing `custom-elements` selector, noticed that the `custom-elements` wrapper isn't present on an individual component page the way it is in a design token's docs